extensions/nodes/nodes.go

```diff
@@ -0,0 +1,114 @@
+package nodes
+
+import (
+	"github.com/rancher/norman/types"
+	v3 "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/kubeapi/secrets"
+	corev1 "k8s.io/api/core/v1"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	initNodeLabelKey         = "rke.cattle.io/init-node"
+	initNodeLabelValue       = "true"
```

Comment:
I think this is not necessary it can be a parse `b, err := strconv.ParseBool("true")` something like this once it gets the label? This would happen in the code snippet I commented below